example/app/javascripts/app.js

```diff
@@ -1,16 +1,28 @@
 var accounts;
-var account;
+var account = _owner;
 var balance;
+accounts[1] = _to;
 
 function setStatus(message) {
   var status = document.getElementById("status");
   status.innerHTML = message;
 };
 
+function initiateToken() {
+  var truf = TruffleToken.deployed();
+
+  var _initialAmount = 10000;
+  var _tokenName = 'Truffle Token';
+  var _decimalUnits = 2;
+  var _tokenSymbol = 'TRFT';
+
+  truf.initiateToken(_initialAmount, _tokenName, _decimalUnits, _tokenSymbol {from: account})
+}
+
 function refreshBalance() {
-  var meta = MetaCoin.deployed();
```

Comment:
Can you rename this variable to a full word? `truffle` would be the obvious choice, but then seems like a misnomer (because it's not Truffle itself). How about `token`?